docker/bootstrap/CHANGELOG.md

```diff
@@ -153,4 +153,8 @@ List of changes between bootstrap image versions.
 
 ## [39] - 2024-12-04
 ### Changes
-- Update build to golang 1.23.4
\ No newline at end of file
+- Update build to golang 1.23.4
+
+## [40] - 2025-01-15
+### Changes
+- Update base image to bookworm
```

Comment:
Since we are changing the bootstrap version, we need to bump the version we use everywhere `39` to `40`. For instance in the Makefile we have `BOOTSTRAP_VERSION=39` which must be changed. Same thing in `test.go` and other places.